### ISSUE TITLE
Silog_loss issue

### ISSUE THREAD

**Issue by Waynechen92** [ENSTA-U2IS/LDU#7]
Hi Xuanlong,

in your loss function silog_loss, you take the log of masked depth estimation and gt. 
However, after masking, there will be many zeros which will result in -inf, how would you tackle this issue?

Best regards,

Wayne

**Comment by xuanlongORZ** [comment]
Hi Wayne,

I think this is not a problem of loss or mask.
The mask is used in depth estimation because not all pixels have ground truth due to data acquisition (e.g., using Lidar). SiLog loss is a widely used loss in monocular depth. For these two parts, I don't think I made some modifications in the implementation compared to the original BTS. 
Normally, mask operation will mask out invalid pixels. The rest pixels should have positive values.
I guess the problem is maybe because of your argument.txt file? You can try to train once on NYU with the original version of BTS.

Best,
Xuanlong

**Comment by Waynechen92** [comment]
Hi Xuanlong,

Thanks for your advice, I found its acutally due to a adjustment I made to your code, I just changed back and it works perfectly now.

Best regards,

Wayne
